### ISSUE TITLE
Select Loss functions to compare

### ISSUE THREAD

**Issue by AdrianArnaiz** [AdrianArnaiz/Brain-MRI-Autoencoder#4]
Select loss functions to be compared for each architecture: i.e. MSE, cross-covariance, dssim, KL divergenci (if VAE), cross-entropy (if semi-supervised), etc

**Comment by AdrianArnaiz** [comment]
* MSE
* SSIM
Big explanation in PDF Report of the Thesis
